src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -204,3 +206,55 @@ void ApplicationViewModel::SetMenuCategories()
     // property setter logic.
     Categories = NavCategoryGroup::CreateMenuOptions();
 }
+
+void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    HandleAlwaysOnTopButtonClick(sender, e);
+}
+
+Concurrency::task<void> ApplicationViewModel::HandleAlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    if (ApplicationView::GetForCurrentView()->GetForCurrentView()->ViewMode == ApplicationViewMode::CompactOverlay)
+    {
+        bool success = co_await ApplicationView::GetForCurrentView()->TryEnterViewModeAsync(ApplicationViewMode::Default);
+        CalculatorViewModel->AreHistoryShortcutsEnabled = success;
+        CalculatorViewModel->HistoryVM->AreHistoryShortcutsEnabled = success;
+        CalculatorViewModel->IsAlwaysOnTop = !success;
+        IsAlwaysOnTop = !success;
+    }
+    else
+    {
+        Windows::Storage::ApplicationDataContainer ^ localSettings = Windows::Storage::ApplicationData::Current->LocalSettings;
```

Comment:
Same comment here and throughout all of the files. in .cpp files the style is to add using namespace Windows::Storage; at the top instead of doing this. Header files should have the fully qualified name though.